packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/node-pool.class.js

```diff
@@ -18,4 +18,8 @@ export default class NodePool {
   isError() {
     return this.status === 'ERROR';
   }
+
+  getMinMaxNodes() {
```

Comment:
Can you rename the function name to better align with returned value like getMinMaxNodesLabel?

Reply:
Yes right.
Fixed.